test/Dockerfile.test.py

```diff
@@ -0,0 +1,22 @@
+# Dockerfile
+FROM python:3.11.3-slim
+
```

Comment:
@kongtiaowang you get those lint errors because flake8 things that this file is a python file. To fix that, you should probably rename it to `Dockerfile.test.python`. That should fix the lint issue.